webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +231,18 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+      .val()
+      .split("4. Nothing happened.\n");
+    // handle the case where stylo pref prepopulated from FF Nightly
+    if (stepsToReproduceStrAr.length > 1) {
```

Comment:
@cch5ng We're always going to enter into this if clause if the user hasn't modified the textarea, right?